.github/workflows/pr-title-checker.yml

```diff
@@ -12,8 +12,8 @@ jobs:
   check:
     runs-on: ubuntu-latest
     steps:
-      - uses: thehanimo/pr-title-checker@v1.3.4
+      - uses: SatelCreative/pr-title-gitmoji-check@1.0.0
         with:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           pass_on_octokit_error: false
-          configuration_path: ".github/pr-title-checker-config.json"
\ No newline at end of file
+          configuration_path: ".github/pr-title-checker-config.json"

```

Comment:
Do we still need this configuration? I thought this was moved to be hardcoded in our pr-title checker.

Reply:
It wasnt but I have moved it now. thanks for catching that